platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/collections/LongListOffHeapTest.java

```diff
@@ -153,6 +153,35 @@ void testPersistShrunkList(final int chunkOffset) throws IOException {
         }
     }
 
+    @Test
+    void updateListCreatedFromSnapshotPersistAndVerify() throws IOException {
+        final int sampleSize = getSampleSize();
+        try (final LongListOffHeap list = createFullyParameterizedLongListWith(
+                sampleSize / 100, // 100 chunks, 100 longs each
+                sampleSize + DEFAULT_NUM_LONGS_PER_CHUNK)) {
+            for (int i = 1; i < getSampleSize(); i++) {
```

Comment:
Should this start at 0?  We're putting from 1-99, but below we're getting from 0 to 99?

Reply:
In this particular case it doesn't make any difference but I changed it for the sake of consistency.